website/content/docs/enterprise/hsm/index.mdx

```diff
@@ -37,5 +37,9 @@ important information on these differences.
 The [Configuration](/docs/configuration/seal/pkcs11) page contains
 configuration information.
 
-Finally, the [Security](/docs/enterprise/hsm/security) page contains
+The [Security](/docs/enterprise/hsm/security) page contains
 information about deploying Vault's HSM support in a secure fashion.
+
+Finally, the [Vault Interoperability Matrix](/docs/interoperability-matrix) page contains
```

Comment:
```suggestion
The [Security](/vault/docs/enterprise/hsm/security) page contains
information about deploying Vault's HSM support in a secure fashion.

Finally, the [Vault Interoperability Matrix](/vault/docs/interoperability-matrix) page contains
```
Hello 👋🏻 Now that thehttps://github.com/hashicorp/vault/pull/18696 has been merged, I'm visiting open PRs to resolve legacy links. All links should now use the full path from the root of developer.hashicorp.com. Let me know if you have any questions!